# header
std_msgs/Header header

# pupil positions
geometry_msgs/Point right_pupil
geometry_msgs/Point left_pupil

# face image
uint32 width
uint32 height
sensor_msgs/Image face_image